icu4c/source/i18n/measunit_impl.h

```diff
@@ -78,13 +89,20 @@ struct SingleUnitImpl : public UMemory {
     /**
      * Simple unit index, unique for every simple unit, -1 for the dimensionless
      * unit. This is an index into a string list in measunit_extra.cpp.
+     *
+     * The default value is -1, meaning the dimensionless unit:
+     * isDimensionless() will return true, until index is changed.
      */
     int32_t index = -1;
 
     /** SI prefix. **/
```

Comment:
Optional: If you add the comment about dimensionality being meaningless, you should probably also say that SI prefix is meaningless

Reply:
Done.